code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -38,15 +38,13 @@
 		if(isliving(usr))
 			var/mob/living/L = usr
 			if(L.electrocute_act(17, src))
-				var/datum/effect_system/spark_spread/s = new /datum/effect_system/spark_spread
-				s.set_up(5, 1, src)
-				s.start()
+				do_sparks(5, 1, src)
 				return 2
 
 	playsound(src.loc, 'sound/machines/click.ogg', 15, 1, -3)
 	for(var/obj/O in src) //Objects
-		O.forceMove(loc)
-	for(var/mob/M in src) //Mobs
+		O.forceMove(loc)

```

Comment:
WAT

Reply:
(Do not worry about this comment I am just trying to comprehend why this was here in the first place)